lib/rails_admin/config/actions/device_ids_mismatch.rb

```diff
@@ -15,8 +15,11 @@ class DeviceIdsMismatch < Base
             @collection = @object.solutions
               .where('solutions.device_id != users.device_id')
               .group('users.id')
-              .pluck('users.device_id', 'users.secret', 'array_agg(solutions.id)')
-              .map { |did, secret, solution_ids| [did, secret, Solution.includes(:task).where(id: solution_ids)] }
+              .pluck('users.device_id', 'users.judge_secret', 'array_agg(solutions.id)')
+              .map do |did, judge_secret, solution_ids|
+              [did, judge_secret,
+                Solution.includes(:task).where(id: solution_ids)]
```

Comment:
If you are going to split it to multiple lines, let's then use full name `device_id` for the first argument.

Also, is the indentation missing here?

Reply:
Split and indentation were done by RuboCop. "did" will rename.

Reply:
done